### DISCUSSION TITLE
New zed website is using a lot of gpu power

### DISCUSSION THREAD

**Discussion by mtsh1000** [zed-industries/zed#16591] [Feature Requests]
New zed website is using a lot of gpu power, could it be a crypto miner?

**Comment by danilo-leal** [comment]
Hey! I'll close this discussion because it's a duplicate of this issue → https://github.com/zed-industries/zed/issues/16563. But no, it was indeed a performance optimization issue on the site that we have recently, hopefully, fixed. Thank you!
